components/datatypes/marketing/commerce.schema.json

```diff
@@ -119,6 +119,22 @@
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "A product has been purchased instantly, potentially skipping the cart or checkout.",
           "meta:descriptionId": "commerce##xdm:instantPurchase##description##9211"
+        },
+        "xdm:requisitionListOpens": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
```

Comment:
Should we have `title` property for any of these or the `meta` properties similar to others in this schema?

Reply:
Good catch about the title. I'll add now, but meta has some unique numbers I am not aware of and seems like it is always added by xdm team during the release.